package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./src/index.js",
   "scripts": {
     "test": "jest",
+    "test-forever": "jest --watchAll",
```

Comment:
Our other repos use:
`"test:watch": "jest --watch --colors"`

We should stay consistent to make our lives easier.

Reply:
okay